lib/irb/cmd/subirb.rb

```diff
@@ -18,45 +18,56 @@ def initialize(conf)
 
       private
 
+      def print_deprecated_warning
+        warn <<~MSG
+          Multi-irb commands are deprecated and will be removed in IRB 2.0.0.
+          If you have any use case for multi-irb, please leave a comment at https://github.com/ruby/irb/issues/653
```

Comment:
LGTM :+1:
I think it will be more helpful if there is a navigation to workspace commands

Reply:
Do you mean something like:

```
If you use the `irb` command to create a new workspace, you can use `chws` instead.
```

Reply:
Yes. or like
```
Multi-irb commands are deprecated and will be removed in IRB 2.0.0. Please use workspace commands instead.
```